sdk/search/Azure.Search.Documents/src/Generated/Models/Tokenizer.cs

```diff
@@ -21,14 +23,13 @@ public Tokenizer()
         /// <param name="name"> The name of the tokenizer. It must only contain letters, digits, spaces, dashes or underscores, can only start and end with alphanumeric characters, and is limited to 128 characters. </param>
         internal Tokenizer(string oDataType, string name)
```

Comment:
While this is internal, "ODataType" should be "odataType" as a parameter name. Not a big deal, but would be nice to handle certain well-known aliases like this (we deal a lot with OData).

Reply:
Hm, should it?

Reply:
Based on https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions, yes. For example, see "IOStream" and "Email". This one falls in between those two, though. You wouldn't name a UInt32 "uInt32", right? Bad example since you should use the alias anyway and not use any sort of Hungarian notation, but I think still illustrates the point.